lib/flipper/route_authorization.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 module Flipper
-  class AdminUserConstraint
+  class RouteAuthorization
```

Comment:
I like the `Constraint` suffix for these types of objects, but I don't feel strongly about it.